<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNewMandato</class>
 <widget class="QWidget" name="CNewMandato">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuovo Mandato</string>
  </property>
  <property name="windowIcon">
   <iconset resource="graphics.qrc">
    <normaloff>:/Resources/Case.PNG</normaloff>:/Resources/Case.PNG</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Numero:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leNumero"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Anno:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leAnno"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Data:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="deDataMandato"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_8">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Banca:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbBanche"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Registrazioni:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout_7">
         <item row="0" column="0">
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Dal:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDateEdit" name="deDal"/>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QFormLayout" name="formLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Al:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDateEdit" name="deAl"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tvRigheMandato">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Importo:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leImporto">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Salva:</string>
       </property>
       <property name="icon">
        <iconset resource="graphics.qrc">
         <normaloff>:/Resources/save-icon.png</normaloff>:/Resources/save-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Chiudi</string>
       </property>
       <property name="icon">
        <iconset resource="graphics.qrc">
         <normaloff>:/Resources/Actions-window-close-icon.png</normaloff>:/Resources/Actions-window-close-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="graphics.qrc"/>
 </resources>
 <connections/>
</ui>
